File: Exemplos/App_IntentDigitalHUB_XamarinAndroid/Elgin_IntentDigitalHUB_XamarinAndroid/IntentServices/Termica/DefinePosicao.cs
namespace Xamarin_Android_Intent_Digital_Hub.IntentServices.Termica
{
    class DefinePosicao : TermicaCommand
    {
        readonly int posicao;

        public DefinePosicao(int posicao) : base("DefinePosicao")
        {
            this.posicao = posicao;
        }

        protected override string FunctionParameters()
        {
            return "\"posicao\"" + ":" + posicao;
        }
    }
}

File: Exemplos M10/App_IntentDigitalHUB_ReactNative/Elgin_IntentDigitalHUB_ReactNative-font/src/intentDigitalHubService/TERMICA/DefinePosicao.ts
import {TermicaCommand} from './TermicaCommand';

export class DefinePosicao extends TermicaCommand {
  readonly posicao: number;

  constructor(posicao: number) {
    super('DefinePosicao');
    this.posicao = posicao;
  }

  functionParameters(): object {
    return {posicao: this.posicao};
  }
}
